### ISSUE TITLE
How to change the target region?

### ISSUE THREAD

**Issue by HeXinsheng16** [paulo-o/forecast_rodeo#1]
If I want to predict the precipitation in another region instead of the western U.S., will this project be applicable?

**Comment by paulo-o** [comment]
Yes, although you would need the corresponding data for the region of interest to run the models. See the SubseasonalRodeo dataset (https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/IHBANG) for the variables used, and Appendix A2 in the paper (https://arxiv.org/abs/1809.07394) for the data sources.

**Comment by HeXinsheng16** [comment]
Thanks for your replying! I've read your paper. It is so excellent that I want to try it in another region. I have downloaded the two dataset (the SubseasonalRodeo dataset and the Reconstructed Precipitation and Temperature CFSv2 Forecasts) and successfully run it in my own computer. But when I want to change the target region, I noticed that the dataset used in the project is much different from the original data. Some of the data only include the data correspond to the target region, some of the data are the principal components. And all the format are transformed into .h5. Therefore, I am confused about how to process the data in order to transform them into the dataset needed here.

**Comment by paulo-o** [comment]
Thanks. The code to process the data sources is not included, so you would have to write your own scripts to load the raw data sources and convert them to Pandas DataFrames that resemble the ones in the SubseasonalRodeo.

**Comment by HeXinsheng16** [comment]
OK, thanks.

**Comment by junjie2008v** [comment]
> Thanks for your replying! I've read your paper. It is so excellent that I want to try it in another region. I have downloaded the two dataset (the SubseasonalRodeo dataset and the Reconstructed Precipitation and Temperature CFSv2 Forecasts) and successfully run it in my own computer. But when I want to change the target region, I noticed that the dataset used in the project is much different from the original data. Some of the data only include the data correspond to the target region, some of the data are the principal components. And all the format are transformed into .h5. Therefore, I am confused about how to process the data in order to transform them into the dataset needed here.

Are you in China?

**Comment by HeXinsheng16** [comment]
> > Thanks for your replying! I've read your paper. It is so excellent that I want to try it in another region. I have downloaded the two dataset (the SubseasonalRodeo dataset and the Reconstructed Precipitation and Temperature CFSv2 Forecasts) and successfully run it in my own computer. But when I want to change the target region, I noticed that the dataset used in the project is much different from the original data. Some of the data only include the data correspond to the target region, some of the data are the principal components. And all the format are transformed into .h5. Therefore, I am confused about how to process the data in order to transform them into the dataset needed here.
> 
> Are you in China?

Yes.

**Comment by augooh** [comment]
> OK, thanks.

Hi! Did you successfully complete to process the data?
